docs/site/Interceptor.md

```diff
@@ -360,6 +360,8 @@ The registration can be further simplified as:
 app.interceptor(MetricsInterceptorProvider, {global: true, group: 'metrics'});
 ```
 
+{% include tip.html content="If you need to intercept all requests, including these destinated to a LB3 mounted app, you can use [a Loopback middleware](https://loopback.io/doc/en/lb4/Middleware.html#loopback-style-middleware)" %}
```

Comment:
Apologies for missing this PR, may I suggest that we reference the Markdown file directly? The docs generator will automatically convert it into the relevant URLs:
```suggestion
{% include tip.html content="If you need to intercept all requests, including these destinated to a LB3 mounted app, you can use a [LoopBack Middleware](./Middleware.md#loopback-style-middleware)" %}
```